superset/assets/javascripts/explorev2/components/ExploreViewContainer.jsx

```diff
@@ -1,16 +1,55 @@
+/* eslint camelcase: 0 */
 import React from 'react';
+import { bindActionCreators } from 'redux';
+import * as actions from '../actions/exploreActions';
+import { connect } from 'react-redux';
 import ChartContainer from './ChartContainer';
 import ControlPanelsContainer from './ControlPanelsContainer';
 import QueryAndSaveBtns from '../../explore/components/QueryAndSaveBtns';
+const $ = require('jquery');
 
-export default class ExploreViewContainer extends React.Component {
+const propTypes = {
+  form_data: React.PropTypes.object.isRequired,
+  actions: React.PropTypes.object.isRequired,
+  slice_id: React.PropTypes.number.isRequired,
+  slice_name: React.PropTypes.string.isRequired,
+  datasource_id: React.PropTypes.number.isRequired,
+  datasource_type: React.PropTypes.string.isRequired,
+};
+
+
+class ExploreViewContainer extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       height: this.getHeight(),
     };
   }
 
+  onQuery() {
+    const data = {};
+    const form_data = this.props.form_data;
+    Object.keys(form_data).forEach((field) => {
+      // filter out null fields
```

Comment:
I added this because there was an error with passing column=null in url params, then it will throw column u'' not found on server side. For checkbox form values it's either true or false, for string form values it can be empty, for arrays we could just not pass in the params and python side has request.args.get['metrics'] or [] to handle these cases.